src/SpectatorView.Unity/Assets/SpectatorView.Editor/Scripts/CompositorWindow.cs

```diff
@@ -239,6 +241,120 @@ private void RecordingGUI()
             }
         }
 
+        private void ColorCorrectionGUI()
+        {
+            colorCorrectionFoldout = EditorGUILayout.Foldout(colorCorrectionFoldout, "Color Correction Settings");
+            if (colorCorrectionFoldout)
+            {
+                RenderTitle("Video Camera Color Correction", Color.clear);
+                CompositionManager compositionManager = GetCompositionManager();
+                bool running = compositionManager != null && compositionManager.TextureManager != null && compositionManager.TextureManager.videoFeedColorCorrection != null;
+                if (running)
+                {
+                    EditorGUILayout.BeginVertical("Box");
+                    {
+                        EditorGUILayout.BeginHorizontal();
+                        {
+                            GUIContent label = new GUIContent("Enabled");
+                            compositionManager.TextureManager.videoFeedColorCorrection.Enabled = EditorGUILayout.Toggle(
+                                label,
+                                compositionManager.TextureManager.videoFeedColorCorrection.Enabled);
+                        }
+                        EditorGUILayout.EndHorizontal();
+                        if (GUI.enabled &&
+                            !compositionManager.TextureManager.videoFeedColorCorrection.Enabled)
+                        {
+                            GUI.enabled = false;
+                        }
+
+                        EditorGUILayout.Space();
+                        EditorGUILayout.BeginHorizontal();
+                        {
+                            GUIContent label = new GUIContent("R Scale");
```

Comment:
>GUIContent label = new GUIContent("R Scale"); [](start = 28, length = 45)

Would be good to have tooltips that describe how these values all manipulate the color, or some documentation about how the ColorCorrection shader works with these values

Reply:
testing these out locally, it seems like adding tool tips to these labels doesn't result in tool tips appearing in the compositor window. I will hold off on this for now.